app/assets/stylesheets/responsive/custom.scss

```diff
@@ -626,35 +626,35 @@ a.link_button_green_large {
 
 .site-title__logo {
   @extend .image-replacement;
-  width: 159px;
+  width: 165px;
   height: 18px;
-  background-size: 159px 18px;
+  background-size: 165px 18px;
   background-image: image-url("wdtk-logo-white.png");
+  // https://css-tricks.com/a-complete-guide-to-svg-fallbacks/
+  background-image: image-url("/assets/wdtk-logo-white.svg"),
```

Comment:
Looks like an incorrect URL – shouldn't need the `/assets/` prefix?

Reply:
oh - no it's wrong for another reason (I think)

Correct me if I'm wrong but `image-url()` doesn't appear to work with SVGs so it should be `url()`, that's why the `/assets/` prefix is there.

Reply:
Nerp – don't think so.

The asset path prefix is configurable, hence the helper.

I _think_ it should just be: `image-url("wdtk-logo-white.svg")` – I don't think the helper really cares about the file extension.

http://guides.rubyonrails.org/v4.2.10/asset_pipeline.html#css-and-sass

Reply:
Yes, that worked, when I tried it before it didn't, which probably means I had a typo in there somewhere. Fixed and pushed